pmlc/target/x86/stencil_tpp_unary.cc

```diff
@@ -61,6 +61,13 @@ class StencilImpl : public pxa::StencilBase {
                            StringRef inName) {
     if (capture)
       return;
+    AffineValueMap ranges = util::getRangesValueMap(op);
```

Comment:
There is code duplication here. These few lines of code seems to repeated multiple times, maybe move them to an utility function?